### PR TITLE
Felt type introduction added

### PR DESCRIPTION
**Pull request by rajeebkm**
- Felt type ```Introduction.md``` updated
- Felt type  ```about.md``` updated

### CODE REVIEW

File: concepts/felts/about.md [hunk 1]
```diff
@@ -1 +1,26 @@
 # Felt Type
+
+# Scalar Types in Cairo
+
+Scalar types represent single values in Cairo. Cairo features three primary scalar types: **felts**, **integers**, and **booleans**. These types are similar to those in other programming languages but have unique characteristics and behaviors in Cairo.
+
+### Felt Type
+
+In Cairo, the default type for a variable or argument, if not explicitly specified, is a **field element**. This is represented by the keyword `felt252`. A field element is an integer within the range `0 ≤ x < `P, where `P` is a very large prime number currently equal to $2^{251} + 17 \cdot 2^{192} + 1$.
```
Comment (misicnenad) [inline concepts/felts/about.md:9]:
```suggestion
In Cairo, the default type for a variable or argument, if not explicitly specified, is a **felt**, also called a **field element**. This is one of the [scalar types](https://book.cairo-lang.org/ch02-02-data-types.html#scalar-types) represented by the keyword `felt252`. A field element is an integer within the range `0 ≤ x < P`, where `P` is a very large prime number currently equal to $2^{251} + 17 \cdot 2^{192} + 1$.
```

File: concepts/felts/about.md [hunk 2]
```diff
@@ -1 +1,26 @@
 # Felt Type
+
+# Scalar Types in Cairo
```
Comment (misicnenad) [inline concepts/felts/about.md:3]: I would remove this "scalar types" section, as we'd like to concept to focus solely on felts. We can just leave link to the "Scalar Types" section of The Cairo Book somewhere in felt's description.

File: concepts/felts/about.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+Scalar types represent single values in Cairo. Cairo features three primary scalar types: **felts**, **integers**, and **booleans**. These types are similar to those in other programming languages but have unique characteristics and behaviors in Cairo.
+
+### Felt Type
```
Comment (misicnenad) [inline concepts/felts/about.md:7]:
```suggestion
## Felt Type
```

File: concepts/felts/about.md [hunk 4]
```diff
@@ -1 +1,26 @@
...
+
+In Cairo, the default type for a variable or argument, if not explicitly specified, is a **field element**. This is represented by the keyword `felt252`. A field element is an integer within the range `0 ≤ x < `P, where `P` is a very large prime number currently equal to $2^{251} + 17 \cdot 2^{192} + 1$.
+
+#### Arithmetic Operations
```
Comment (misicnenad) [inline concepts/felts/about.md:11]:
```suggestion
## Arithmetic Operations
```

File: concepts/felts/about.md [hunk 5]
```diff
@@ -1 +1,26 @@
...
+
+When performing arithmetic operations (addition, subtraction, multiplication) with field elements, if the result falls outside the range of the prime number `P`, an overflow or underflow occurs. To handle this, Cairo adjusts the result by adding or subtracting an appropriate multiple of `P` to bring the result back within the valid range. Essentially, this means the result is computed modulo `P`.
+
+#### Division in Cairo
```
Comment (misicnenad) [inline concepts/felts/about.md:15]:
```suggestion
## Division in Cairo
```

File: concepts/felts/about.md [hunk 6]
```diff
@@ -1 +1,26 @@
...
+
+A critical distinction between integers and field elements is how division is handled. Unlike regular CPU integer division, where the result of `x` divided by `y` is defined as the integer part of the quotient (returning the floor value), Cairo ensures that the division of field elements always satisfies the equation:
+
+$(x / y) * y = x$
```
Comment (misicnenad) [inline concepts/felts/about.md:19]: Can you make all these divisions appear as fractions, similar to how you've done on [line 23](https://github.com/exercism/cairo/pull/56/files#diff-598032b99d9fffc7e22fbe18fe245d4c88c88a0044b7ba60b89b3688666dcab8R23) (the `\frac{P+1}{2}` part).

File: concepts/felts/about.md [hunk 7]
```diff
@@ -1 +1,26 @@
 # Felt Type
```
Comment (misicnenad) [inline concepts/felts/about.md:1]: Please update the _.meta/config.json_ file in the following way:
- remove the existing author (`misicnenad`) and add your GH handle
- add the missing [blurb](https://exercism.org/docs/building/tracks/concepts#h-documentation-files)

**Review by misicnenad — CHANGES_REQUESTED** [review]
The concept looks good, left some small change requests

**Review by rajeebkm — COMMENTED** [review]
- Change request resolved.

**Review by misicnenad — APPROVED** [review]
This looks beautiful, thanks!

**Pull request merged**